src/passes/abiExtractor.ts

```diff
@@ -31,6 +31,7 @@ export class ABIExtractor extends ASTMapper {
     );
     node.vContracts
       .filter((c) => !c.abstract)
+      .flatMap((cd) => cd.vLinearizedBaseContracts)
```

Comment:
This won't introduce duplicates since we're using canonical signatures and a set. Also if a contract is abstract and inherited the overriden contract's function's visibility can't be modified.